src/onnx/onnx.cpp

```diff
@@ -490,6 +496,9 @@ struct onnx_parser
         {
             axis = parse_value(attributes.at("axis")).at<int>();
         }
+        args[0] = make_contiguous(args[0]);
+        args[1] = make_contiguous(args[1]);
+
         op::gather op{axis};
         return prog.add_instruction(op, std::move(args));
```

Comment:
Can we just do `prog.add_instruction(op, make_contiguous(args[0]), make_contiguous(args[1]))` here?

Reply:
changed accordingly